rustls/src/client/hs.rs

```diff
@@ -741,15 +742,15 @@ impl State for ExpectServerHello {
                 let certv = verify::ServerCertVerified::assertion();
                 let sigv = verify::HandshakeSignatureValid::assertion();
 
-                return if self.must_issue_new_ticket {
+                return if must_issue_new_ticket {
                     Ok(self.into_expect_tls12_new_ticket_resume(secrets, certv, sigv))
                 } else {
                     Ok(self.into_expect_tls12_ccs_resume(secrets, certv, sigv))
                 };
             }
         }
 
-        Ok(self.into_expect_tls12_certificate(scs))
+        Ok(self.into_expect_tls12_certificate(scs, may_send_cert_status, must_issue_new_ticket, server_cert_list_list))
```

Comment:
The danger here is that `may_send_cert_status` and `must_issue_new_ticket` are both `bool`. We're relying on the state machine tests to make sure we don't accidentally transpose them in this call.